content/applications/finance/accounting/payments.rst

```diff
@@ -30,114 +32,149 @@ At this point, the customer invoice or vendor bill is marked as :guilabel:`In pa
 the outstanding account is reconciled with a bank statement line, the invoice or vendor bill changes
 to the :guilabel:`Paid` status.
 
-The information icon near the payment line displays more information about the payment. You can
-access additional information, such as the related journal, by clicking :guilabel:`View`.
+The :icon:`fa-info-circle` information icon next to the payment line displays more
+information about the payment. To access additional information, such as the related journal, click
+:guilabel:`View`.
 
 .. image:: payments/information-icon.png
    :alt: See detailed information of a payment.
 
 .. note::
    - The customer invoice or vendor bill must be in the :guilabel:`Posted` status to register the
      payment.
-   - If you unreconcile a payment, it still appears in your books but is no longer linked to the
+   - If a payment is unreconciled, it still appears in the books but is no longer linked to the
      invoice.
-   - If you (un)reconcile a payment in a different currency, a journal entry is automatically
+   - If a payment is (un)reconciled in a different currency, a journal entry is automatically
      created to post the currency exchange gains/losses (reversal) amount.
-   - If you (un)reconcile a payment and an invoice having cash basis taxes, a journal entry is
-     automatically created to post the cash basis tax (reversal) amount.
+   - If a payment is (un)reconciled and an invoice with cash-basis taxes, a journal entry is
+     automatically created to post the cash-basis tax (reversal) amount.
 
 .. tip::
-   - If your main bank account is set as an :ref:`outstanding account
+   - If the main bank account is set as an :ref:`outstanding account
      <bank/outstanding-accounts>`, and the payment is registered in Odoo (not through a related
      bank statement), invoices and bills are directly registered as :guilabel:`Paid`.
```

Comment:
Let's take this opportunity to update this language to be about bank **transactions** instead of statements. Before 16, you couldn't upload individual transactions - they had to be part of bank statements, and each transaction was called a "bank statement line". Now, they are by default individual transactions that can be grouped into statements.